src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.exception.InvalidInputDataException;
+import core.basesyntax.service.exception.UserAlreadyExistsException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword123";
+    private static final Integer VALID_AGE = 33;
+    private static final Integer NEGATIVE_AGE = -1;
+    private static final Integer AGE_LESS_THAN_18 = 17;
+    private static final String SHORT_LOGIN = "short";
+    private static final String SHORT_PASSWORD = "pass5";
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
```

Comment:
As AI Mentor mentioned, you should use `@BeforeAll` for initializing RegistrationServiceImpl.

![image](https://github.com/user-attachments/assets/d6ce71c1-9080-4287-9d97-3e6289987560)